src/extension/ChronoController.hpp

```diff
@@ -37,8 +37,10 @@ namespace extension {
                 // Lock the mutex while we're doing stuff
                 const std::lock_guard<std::mutex> lock(mutex);
 
-                // Add our new task to the heap
-                tasks.push_back(*task);
+                // Add our new task to the heap if we are still running
+                if (running) {
```

Comment:
Would powerplants running flag work for this? or is there an advantage to having our own

Reply:
I added our own because IOController has its own :shrug: